index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'zukijs';
```

Comment:
![image](https://github.com/Sabsterrexx/zukiJS/assets/98546495/efd746c3-8545-47a6-93d5-0271e8852f9b)
this won't go away without this line. it's incredibly annoying and users _will_ notice.